### ISSUE TITLE
Clarify how to handle Submissions if the pathing leads to a multi-credential envelope format

### ISSUE THREAD

**Issue by csuwildcat** [decentralized-identity/presentation-exchange#108]
If you path to a VP, talk about how to iterate the creds inside against the descriptor.

**Comment by csuwildcat** [comment]
```jsonld=
}
  "presentation_submission": {
    "descriptor_map": [
      { // This 
        "id": "banking_input_2",
        "paths": [
          {
            "format": "jwt_vp",
            "path": "$.things.[0]"
          },
          {
            "format": "jwt_vc",
            "path": "$.things.[0].verifiableCredentials.[0]"
          }
        ]
      },
```

**Comment by glcohen** [comment]
What about mirroring `from` and `from_nested` in the request to have `path` and `path_nested`. This would also allow the nesting to represent the shape you wish to parse.
```
{
  "presentation_submission": {
    "descriptor_map": [
      { 
        "id": "banking_input_2",
        "format": "jwt_vp",
        "path": "$.outer.[0]"
        "path_nested": {
            "format": "ldp_vc",
            "path": "$.inner.[1]",
            "path_nested": {
                "format": "jwt_vc",
                "path": "$.superinner.[2]"
            }
        }
    }
  ]
}
```

**Comment by glcohen** [comment]
Resolved in #125
